server/conf/database.properties.tmpl

```diff
@@ -9,10 +9,10 @@ scalar.db.username={{ default .Env.SCALAR_DB_USERNAME "" }}
 scalar.db.password={{ default .Env.SCALAR_DB_PASSWORD "" }}
 
 # Storage implementation. "cassandra" or "cosmos" or "dynamo" or "jdbc" or "grpc" can be set. Default storage is "cassandra".
-scalar.db.storage={{ default .Env.SCALAR_DB_STORAGE "cassandra" }}
+scalar.db.storage={{ default .Env.SCALAR_DB_STORAGE "" }}
 
 # The type of the transaction manager. "consensus-commit" or "jdbc" or "grpc" can be set. The default is "consensus-commit"
-scalar.db.transaction_manager={{ default .Env.SCALAR_DB_TRANSACTION_MANAGER "consensus-commit" }}
+scalar.db.transaction_manager={{ default .Env.SCALAR_DB_TRANSACTION_MANAGER "" }}
 
 # Default isolation level for ConsensusCommit. Either SNAPSHOT or SERIALIZABLE can be specified. SNAPSHOT is used by default.
 scalar.db.consensus_commit.isolation_level={{ default .Env.SCALAR_DB_CONSENSUSCOMMIT_ISOLATION_LEVEL "" }}
```

Comment:
I just realized that the GraphQL uses `CONSENSUS_COMMIT`, but not `CONSENSUSCOMMIT`. We should unify them?

Reply:
I may have changed this naming regardless of the purpose of that PR. https://github.com/scalar-labs/scalardb-graphql/pull/20/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L31-R30

Changing this variable name would introduce incompatibility, so we should probably keep the name `CONSENSUSCOMMIT`.

Reply:
I have reverted back the naming to `CONSENSUSCOMMIT` in the scalar-labs/scalardb-graphql#20 side.

Reply:
I see. Thank you!
(I found an another issue of environment variable in Dockerfile of Scalar DB. It is not related to this PR. I will fix it in the another PR.)

Reply:
> (I found an another issue of environment variable in Dockerfile of Scalar DB. It is not related to this PR. I will fix it in the another PR.)

Sorry, it's my misunderstanding...
We don't need to fix.